audit_log/settings.py

```diff
@@ -0,0 +1,53 @@
+import re
+
+from django.conf import settings
+from django.core.signals import setting_changed
+from django.dispatch import receiver
+
+_defaults = dict(
+    ENABLED=True,
+    ORIGIN="service",
+    LOGGED_ENDPOINTS_RE=re.compile(r"^/(v1|gdpr-api)/"),
+    REQUEST_AUDIT_LOG_VAR="_audit_logged_object_ids",
+    LOG_TO_DB_ENABLED=True,
+    LOG_TO_LOGGER_ENABLED=False,
+)
```

Comment:
Putting these into `settings.py` like for other environment variables would make environment variables perhaps be handled in a more centralized / same way. Just thinking is there value in having these here instead of in `settings.py`. How do you see this?

Reply:
Ok, we discussed this and the value here is that the audit log functionality is supposed to be splittable to its own package, so having its own settings here can be argued to be justified because of this.